README.md

```diff
@@ -0,0 +1,62 @@
+OpenSSL Technical Policies
+==========================
+
+This repository contains the technical policies and procedures established by
+the [OTC] in accordance with the [project bylaws] and the requirements specified
+by the [OMC].
+
+
+The policies
+------------
+
+The policies are stored in the [policies](./policies) subdirectory, each in
+a separate markdown file called `policy-name.md` where `policy-name` is a short
+memorable description of the policy such as `voting-procedure`, `design-process`,
+or similar.
+
+New policies or changes to existing policies are proposed and discussed in pull
+requests and issues of this repository, and finally merged after an approval by
+an OTC vote. The details are regulated by the [policy-change-process] and the
+[voting-procedure] policy.
+
+[policy-change-process]: ./policies/policy-change-process.md
+[voting-procedure]: ./policies/voting-procedure.md
+
+
+The voting records
+------------------
+
+Each formal vote of the OTC is stored in the [votes](./votes) subdirectory in a
+separate file.
+
+The voting records are named `vote-yyyy-mm-dd-vote-name.txt` where the `yyyy-mm-dd`
```

Comment:
The filename format is mandated by the voting-procedure. You cannot change this format without changing the procedure first.

Reply:
Good point. Looking at [voting-procedure](https://github.com/openssl/technical-policies/blob/master/policies/voting-procedure.md) document, I noticed that it actually does not make sense to duplicate the information about the file format names and file format hier. I'll replace it with a link to the policy.

Reply:
And I'll revert the renaming. It should be done in sync with an update to the policy in a separate pr.

Reply:
Done. See https://github.com/mspncp/technical-policies/tree/add-technical-policies-readme to view the rendered markdown.